nyc_taxis/README.md

```diff
@@ -74,6 +74,9 @@ This workload allows to overwrite the following parameters using `--workload-par
 * `target_throughput` (default: default values for each operation): Number of requests per second, `none` for no limit.
 * `search_clients`: Number of clients that issues search requests.
 
+### Test Procedures
+The workload contains different test procedures, see [TEST_PROCEDURES](TEST_PROCEDURES.md) for details.
```

Comment:
This workload contains multiple test procedures, ...

Reply:
Hi @gkamat, thanks for your careful review! 👍 I will make the changes.

Reply:
Updated the wording according to your suggestions in a new commit https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/58/commits/c8cfc9d7953875a8e2b539191443b477d247b5ea. Thank you!